cmd/neofs-node/httpcomponent.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	httputil "github.com/nspcc-dev/neofs-node/pkg/util/http"
+)
+
+type httpComponent struct {
```

Comment:
is it possible to move it to `pkg`?

also making it a separate public component could help with parametrizing it like `logger` and implementing `Reload` method i guess

Reply:
I thought about `Reload` when reviewed `neofs-ir` - in ir we also have pprof and metrics. Should we move it to pkg in this PR?